app/controllers/users.js

```diff
@@ -1,15 +1,19 @@
 const userService = require('../services/users');
+const { databaseError } = require('../errors');
+const errorMsg = require('../constants/errorMessages');
+const varParse = require('../serializers/varParse');
 
 exports.signUp = async (req, res) => {
   try {
-    const userCreated = await userService.createUser({
+    const userCreated = await varParse.nameParse({
       firstName: req.body.first_name,
       lastName: req.body.last_name,
       email: req.body.email,
       password: req.body.password
     });
+    userService.createUser(userCreated);
     return res.status(201).send(userCreated);
   } catch (error) {
-    return res.status(500).send(error);
+    throw databaseError(errorMsg.userError);
   }
```

Comment:
Remember hot to we manage the error in the controller file.
See the jokes controller in catch block.